### DISCUSSION TITLE
如何修改iOS/iPadOS Safari浏览器标题栏颜色？

### DISCUSSION THREAD

**Discussion by wozsun** [vuepress-theme-hope/vuepress-theme-hope#3239] [Q&A] [answered]
<img width="1194" alt="image" src="https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/80951424/6ecd7b58-8a2e-493d-90b8-d1548d45794a">
如图示，标题栏的颜色似乎不受主题色的控制，目前已知可以在head中添加["meta", { name: "theme-color", content: "color" }]来实现单种颜色的控制，那么如何才能让标题栏颜色和主题色一致？

**Comment by Mister-Hope** [comment, marked as answer]
写个js监测一下html的class吧.....

这东西没必要实现，因为pwa之类的功能本来也不支持动态主题色。另外我非常不确定你动态调整meta theme-color有效果，有些浏览器可能只在加载网页的时候读一次。

**Reply by wozsun** [reply]
明白了，感谢，我去试一试
